shielddash/core/static/js/app/api/login-api.js

```diff
@@ -0,0 +1,42 @@
+import store from '../store';
+import {getLoginSuccess, getLogoutSuccess} from '../actions/login-actions';
+import {browserHistory} from  'react-router';
+
+
+export function performSignIn() {
+  var user = {};
+
+  // Already signed in?
+  if (localStorage.token) {
+    user.name = localStorage.name;
+    user.token = localStorage.token;
+    store.dispatch(getLoginSuccess(user));
+    console.log('...already signed in');
+  } else { // Not signed in. Do the Google OAuth dance.
+    window.auth2.signIn().then(function() {
+      user.name = auth2.currentUser.get().getBasicProfile().getName();
+      user.email = auth2.currentUser.get().getBasicProfile().getEmail();
+      user.token = gapi.auth2.getAuthInstance().currentUser.get().getAuthResponse().id_token;
+      localStorage.setItem('user_name', user.name);
+      localStorage.setItem('user_email', user.email);
+      localStorage.setItem('user_token', user.token);
```

Comment:
We probably want/need a timeout on these b/c the token will time out. I'm not sure how long they last tho. Can we file an issue to look at this later?